src/BitmapText.cpp

```diff
@@ -672,8 +686,45 @@ bool BitmapText::EarlyAbortDraw() const
 	return m_wTextLines.empty();
 }
 
+namespace {
```

Comment:
Let's put anonymous namespaces at the top of the files instead of inbetween member function definitions.